weather-app/app.js

```diff
@@ -0,0 +1,12 @@
+const { getGeoData } = require('./helpers/geoData')
+const { getWeather } = require('./helpers/weatherData')
+
+const location = "belarus"
```

Comment:
Please try to pass empty string. Also you removed `package.json` file - i can't install dependencies !!